models/iris_depth_camera_3/model.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<model>
+  <name>3DR Iris with 3 depth cmaera</name>
+  <version>1.0</version>
+  <sdf version='1.4'>iris_depth_camera_3.sdf</sdf>
+  <author>
+   <name>Lorenz Meier and Thomas Gubler</name>
+   <email>lorenz@px4.io</email>
+  </author>
+  <description>
+    This is a model of the 3DR Iris Quadrotor with 3 depth cameras.
```

Comment:
Ok so the main difference between `iris_depth_camera` and `iris_depth_camera_3` are the number of cameras? What about `iris_triple_depth_camera`? Seems more suited than `iris_depth_camera_3`, as it the `_3` suffix seems like it's version 3 of `iris_depth_camera`.